setup.py

```diff
@@ -810,10 +810,13 @@ def run(self):
             'borgfs = borg.archiver:main',
         ]
     },
+    # See also the MANIFEST.in file.
+    # We want to install all the files in the package directories...
     include_package_data=True,
-    package_data={
-        'borg': ['paperkey.html'],
-        'borg.testsuite': ['attic.tar.gz'],
+    # ...except the source files which have been compiled (C extensions):
+    exclude_package_data={
+        '': ['*.c', '*.h', '*.pyx', ],
+        'borg.algorithms': ['blake2/README.md', 'blake2/COPYING', ],
```

Comment:
IANAL, but shouldn't the blake2 license be mentioned somewhere if the elements are excluded for wheels/install

Reply:
guess so, yes. but as dealing with license files is not in the scope of this PR, I'll just remove that for now, so it can be dealt with in a separate PR.